### PR TITLE
Feature/908 ee events

### PR DESCRIPTION
**Pull request by kevinhowbrook**
Implements ticket [908](https://projects.torchbox.com/projects/rca-website-rebuild/tickets/908) related events in commit https://projects.torchbox.com/projects/rca-website-rebuild/tickets/908

### CODE REVIEW

File: rca/landingpages/migrations/0028_add_related_events.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Generated by Django 3.1.10 on 2021-07-26 07:40
```
Comment (richbrennan) [inline rca/landingpages/migrations/0028_add_related_events.py:1]: Is it possible to reduce these to one migration?

File: rca/landingpages/models.py [hunk 2]
```diff
@@ -560,9 +564,112 @@ class Meta:
...
+        "landingpages.EELandingPage", related_name="related_editorial_pages"
+    )
+    panels = [PageChooserPanel("page", ["editorial.EditorialPage"])]
+
+
+class EELandingPageRelatedEventPage(RelatedPage):
+    source_page = ParentalKey(
+        "landingpages.EELandingPage", related_name="related_event_pages"
+    )
+    panels = [PageChooserPanel("page", ["events.EVentDetailPage"])]
```
Comment (richbrennan) [inline rca/landingpages/models.py:578]: Capital `V` in `EVentDetailPageEVentDetailPage`

**Review by richbrennan — COMMENTED** [review]
A couple of comments, thanks Kevin.

File: rca/landingpages/migrations/0028_add_related_events.py (hunk 1 above)
Comment (kevinhowbrook) [reply rca/landingpages/migrations/0028_add_related_events.py:1]: Seen as all these are split tickets it's just easier to follow 1 migrations per PR for me

**Pull request merged**